Source/WebCore/crypto/mac/CryptoKeyOKPCocoa.cpp

```diff
@@ -272,6 +289,9 @@ RefPtr<CryptoKeyOKP> CryptoKeyOKP::importPkcs8(CryptoAlgorithmIdentifier identif
             if (keyData[index++] != OKPOIDEd25519Byte)
                 return nullptr;
             break;
+        default:
+            ASSERT_NOT_REACHED();
+            break;
```

Comment:
This case should also avoid further execution and return nullptr.

Reply:
The function returns void and there is no further code to execute after this, so maybe not needed ?

Reply:
Unless the GitHub UI is broken, I think this one returns `RefPtr<CryptoKeyOKP>`, so it should return nullptr. But `writeOID()` above does return void and there's nothing else after that switch statement, so a break there is fine.

Reply:
oh, true, I totally misread it :( 
Will fix it now.